airbyte-integrations/connectors/source-trustpilot/unit_tests/test_data/data.py

```diff
@@ -0,0 +1,6 @@
eplies","business-profile-page/filters/replies/body":"Reviews with a reply from a company.","business-profile-page/filters/header/total-reviews/zero":"0 total","business-profile-page/filters/header/total-reviews/one":"1 total","business-profile-page/filters/header/total-reviews/many":"[COUNT] total","business-profile-page/filters/no-reviews/header":"No reviews","business-profile-page/filters/no-reviews/body":"This company hasn’t received any reviews yet.","business-profile-page/filters/no-matches/header":"No matches","business-profile-page/filters/no-matches/body":"Try changing or removing some of your filters.","business-profile-page/filters/button/reset":"Reset","business-profile-page/filters/button/filter":"Filter","business-profile-page/filters/modal/header":"Filter by","business-profile-page/filters/modal/rating":"Rating","business-profile-page/filters/modal/recommended":"Recommended","business-profile-page/filters/modal/date-posted":"Date posted","business-profile-page/filters/modal/date-posted/all-reviews":"All reviews","business-profile-page/filters/modal/date-posted/last-30-days":"Last 30 days","business-profile-page/filters/modal/date-posted/last-3-months":"Last 3 months","business-profile-page/filters/modal/date-posted/last-6-months":"Last 6 months","business-profile-page/filters/modal/date-posted/last-12-months":"Last 12 months","business-profile-page/filters/modal/popular-mentions":"Popular mentions","business-profile-page/filters/modal/search":"Search reviews","business-profile-page/filters/modal/search/placeholder":"Search by keyword","business-profile-page/filters/modal/language":"Language","business-profile-page/filters/modal/show-more":"Show more","business-profile-page/filters/modal/button/show":"Show reviews","business-profile-page/filters/modal/button/show/zero":"Show 0 reviews","business-profile-page/filters/modal/button/show/one":"Show 1 review","business-profile-page/filters/modal/button/show/many":"Show [COUNT] reviews","business-profile-page/review-sort/modal/header":"Sort by","business-profile-page/review-sort/dropdown/default":"Sort","business-profile-page/review-sort/label":"Sort by:","business-profile-page/review-sort/types/recency":"Most recent","business-profile-page/review-sort/types/relevance":"Most relevant","business-profile-page/review-sort/help/title":"Sorting reviews","business-profile-page/review-sort/help/body":"Reviews on Trustpilot can be sorted by recency or by relevance.[DOUBLE-LINE-BREAK]Reviews are considered relevant if they’re newer and have longer, more detailed content. [LINK-BEGIN]Learn more[LINK-END]","business-profile-page/header/business-information/visit-company-button":"Visit this website","business-profile-page/header/business-information/verified-company":"VERIFIED COMPANY","business-profile-page/header/business-information/verified-company/modal/header":"Verified additional company details","business-profile-page/header/business-information/verified-company/modal/body":"Transparency is i
```

Comment:
why you're using the html response and not the json one?

Reply:
also can you format this file?

Reply:
Because this source is based on HTML parsing, not direct API

Reply:
Formated HTML